### DISCUSSION TITLE
[質問] riverpod_graph はどうやって使いましたか？

### DISCUSSION THREAD

**Discussion by JAICHANGPARK** [mono0926/wdb106-flutter#6] [Q&A] [answered]
タイトルの通りですが、
riverpod_graphをどうやって使って、
使い方が気になります。

ありがとうございます。

**Comment by mono0926** [comment, marked as answer]
未リリースな現状ですが、 https://twitter.com/_mono/status/1520946443254923264 に記載のやり方で実行できます🐶
